dbms/src/Encryption/EncryptionPath.h

```diff
@@ -14,19 +14,43 @@
 
 #pragma once
 
-#include <string>
+#include <common/types.h>
+#include <pingcap/pd/IClient.h>
```

Comment:
```suggestion
#include <pingcap/pd/Types.h>
```

Change it and `tiflash_common_io` should be able to not rely on `kv_client` now

Reply:
done